### PR TITLE
Add endpoint to get all unverified tests, assigned to current coach

### PR DESCRIPTION
**Pull request by maxim12321**

### CODE REVIEW

File: src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java [hunk 1]
```diff
@@ -69,6 +69,11 @@ public Test getById(long id) {
         return testsRepository.getByStatuses(statuses);
     }
 
+    @Override
+    public List<Test> getAllUnverifiedTestsByCoach(long coachId) {
+        return testsRepository.getAllByAssignedCoachAndStatus(coachId, Status.COMPLETED);
```
Comment (AntonMaksimenkau) [inline src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: in-verification?

**Review by AntonMaksimenkau — CHANGES_REQUESTED** [review]

File: src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java (hunk 1 above)
Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: This page is for coaches to start test verification. The page displays all **unverified** tests, that were assigned to the current user.

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: So, when the status is `in-verification`, the test shouldn't be displayed here)

Comment (AntonMaksimenkau) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: They should. The coach could leave a page and should have ability to continue the verification

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: Oh, yeah. I'll rename the method.

Comment (maxim12321) [reply src/main/java/com/team4/testingsystem/services/impl/TestsServiceImpl.java]: Oh, maybe I don't need to rename it :)

**Review by AntonMaksimenkau — APPROVED** [review]

**Review by Anlord241 — APPROVED** [review]

**Review by Darina007 — APPROVED** [review]

**Pull request merged**